src/components/Code/Code.scss

```diff
@@ -1,76 +1,74 @@
 @import "./../../base/base.scss";
 @import "../Input/Input.scss";
 
-.n-code-main {
-  width: 100%;
-  position: relative;
-}
-
 .n-code-field-container {
   display: flex;
   width: 100%;
   justify-content: space-between;
-  z-index: 1;
-  position: relative;
-}
-
-.n-code-4 {
-  width: calc(100% / 6);
-}
-
-.n-code-6 {
-  width: calc(100% / 8);
+  gap: 10px;
```

Comment:
Recommendation would be to use rem in place of px.

Reply:
will do